### PR TITLE
Prompt with project keywords 😎

### LINKED ISSUE #87: Add keywords prompt to end up in package.json
I just noticed all my bootstraped projects with `generator-git` have the "keywords" section empty!

I think its an easy fix and can make a PR later to fix this as well 😉 

Happy hacktoberfest! 🍻

### PR DESCRIPTION
**Pull request by kutyel**
Closes #87

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -88,6 +91,7 @@ module.exports = class extends Generator {
 
     promise.then(props => {
       this.appDescription = props.appDescription
+      this.keywords = props.keywords.trim().split(',')
```
Comment (Kikobeats) [inline app/index.js]: better use space for separation

Comment (Kikobeats) [reply app/index.js]: Also keep in mind what happens if the user don't enter nothing

Comment (kutyel) [reply app/index.js]: Are you sure you want spaces? I've seen may projects with keywords like `functional programming` and they always end up split. 😕

Comment (Kikobeats) [reply app/index.js]: I'm not sure 🤔 https://github.com/npm/npm/issues/7072

**Comment by kutyel** [comment]
Done 😎

**Pull request merged**